src/prop/cnf_stream.h

```diff
@@ -203,6 +203,13 @@ class CnfStream {
   SatLiteral handleAnd(TNode node);
   SatLiteral handleOr(TNode node);
 
+  /** Stores the literal of the given node in d_literalToNodeMap.
+   *
+   * Note that n must already have a literal associated to it in
+   * d_nodeToLiteralMap.
+   */
+  void ensureExistingLiteral(TNode n);
```

Comment:
To avoid confusion with ensureLiteral, let's call this `ensureMappingForLit` or something along these lines.  Also can this be private (ProofCnfStream is friend?).

Reply:
Currently there are no private components to CnfStream, just protected ones. Indeed `ProofCnfStream` is a friend. I'd say we can change this in another PR.